### ISSUE TITLE
`GET https://graph.microsoft.com/v1.0/me` resulted in a `404 Not Found`

### ISSUE THREAD

**Issue by samuelsantaera** [microsoftgraph/msgraph-sdk-php#382]
Hi everyone,
how is it possible this? 
I've been trying to work something for days but always without success.
Someone can help me? 
[AB#8037](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8037)

**Comment by zengin** [comment]
@samuelsantaera, could you please provide more details? What did you use to hit the URL: `https://graph.microsoft.com/v1.0/me`? Is it the PHP SDK? If so, could you please share your code?

**Comment by samuelsantaera** [comment]
Thanks but I solved it, I was using an incorrect authentication procedure, the one shown in the README.md file but it didn't work.
https://docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/graph-oauth?view=odsp-graph-online
I solved using the procedure "Code flow" linked perché porca boia non si capisce un culo di come devi fare a collegarti at Graph.
The documentation is too confusing.
